astropy/config/paths.py

```diff
@@ -239,7 +231,7 @@ class set_temp_config(_SetTempPath):
     """
 
     _directory_type = "config"
-    _directory_env_var = "XDG_CONFIG_HOME"
+    _xdg_env_dir = "XDG_CONFIG_HOME"
```

Comment:
I don't think this rename is in scope here either. Makes the diff even harder to read.

Reply:
IMO the pre-existing name becomes ambiguous with this patch, so I'd like to keep the rename. But if you feel strongly about it it can also be isolated in a separate PR